Multisig/UI/ClaimToken/ReviewClaimSafeTokenTransactionViewController.swift

```diff
@@ -58,7 +55,19 @@ class ReviewClaimSafeTokenTransactionViewController: ReviewSafeTransactionViewCo
     }
 
     override func createTransaction() -> Transaction? {
-        SafeTransactionController.shared.claimTokenTransaction(safe: safe, safeTxGas: safeTxGas, nonce: nonce)
+        let delegateAddress = selectedCustomAddress ?? selectedGuardian?.address.address
```

Comment:
I think for safer code we need to check that one of those two parameters has a value

Reply:
👍 added to the safe token flow